doc/nrf/ug_matter_creating_accessory.rst

```diff
@@ -187,13 +237,14 @@ To add a new timer for the measurement event, edit the :file:`src/app_task.cpp`
            return 0;
    }
 
-If :c:func:`StartSensorTimer()` is called, the :c:struct:`SensorMeasure` event is added to the event queue every *aTimeoutMs* milliseconds, until :c:func:`StopSensorTimer()` is called.
+The timer must be initialized in the ``Init()`` method of the ``AppTask`` class.
+If :c:func:`StartSensorTimer()` is called, which points to the ``SensorTimer`` object, the :c:struct:`SensorMeasure` event is added to the event queue every *aTimeoutMs* milliseconds, until :c:func:`StopSensorTimer()` is called.
```

Comment:
I'm not sure what `which points to the SensorTimer object` means here. Did I say sth like that in the recording? :)